client/app/iplb/serverFarm/probe/iplb-server-farm-probe.html

```diff
@@ -1,103 +1,77 @@
+                data-min="1"
+                data-max="65535">
+            </oui-numeric>
+        </oui-field>
 
-            <div class="oui-field"
-                data-ng-if="IpLoadBalancerServerFarmProbeEditCtrl.isFieldVisible('match')">
-                <label for="probe_match" class="oui-label" data-translate="iplb_farm_edit_probe_match"></label>
-                <div class="oui-field__content">
-                    <oui-radio-toggle-group data-model="IpLoadBalancerServerFarmProbeEditCtrl.farmProbe.match">
-                        <oui-radio data-ng-repeat="match in IpLoadBalancerServerFarmProbeEditCtrl.getMatches() track by $index"
-                            data-ng-hide="IpLoadBalancerServerFarmProbeEditCtrl.farmProbe.type !== 'http' && match === 'status'"
-                            data-text="{{ ::('iplb_farm_edit_probe_match_' + match) | translate }}"
-                            data-value="match">
-                        </oui-radio>
-                    </oui-radio-toggle-group>
-                </div>
-            </div>
+        <oui-field data-label="{{ ::'iplb_farm_edit_probe_method' | translate }}" data-ng-if="IpLoadBalancerServerFarmProbeEditCtrl.isFieldVisible('method')">
+            <oui-radio-toggle-group data-model="IpLoadBalancerServerFarmProbeEditCtrl.farmProbe.method">
+                <oui-radio
+                    data-text="{{ ::('iplb_farm_edit_probe_method_' + method) | translate }}"
+                    data-value="method"
+                    data-ng-repeat="method in IpLoadBalancerServerFarmProbeEditCtrl.methods track by $index">
+                </oui-radio>
+            </oui-radio-toggle-group>
+        </oui-field>
 
-            <div class="oui-field"
-                data-ng-if="IpLoadBalancerServerFarmProbeEditCtrl.isFieldVisible('pattern')">
-                <label for="probe_pattern" class="oui-label" data-translate="iplb_farm_edit_probe_pattern"></label>
-                <div class="oui-field__content">
-                    <div class="oui-field-control">
-                        <input type="text" id="probe_pattern" name="probe_pattern" class="oui-input" data-ng-model="IpLoadBalancerServerFarmProbeEditCtrl.farmProbe.pattern" />
-                    </div>
-                </div>
-            </div>
+        <oui-field data-label="{{ ::'iplb_farm_edit_probe_match' | translate }}" data-ng-if="IpLoadBalancerServerFarmProbeEditCtrl.isFieldVisible('match')">
+            <oui-radio-toggle-group data-model="IpLoadBalancerServerFarmProbeEditCtrl.farmProbe.match">
+                <oui-radio
+                    data-text="{{ ::('iplb_farm_edit_probe_match_' + match) | translate }}"
+                    data-value="match"
+                    data-ng-repeat="match in IpLoadBalancerServerFarmProbeEditCtrl.getMatches()"
+                    data-ng-hide="IpLoadBalancerServerFarmProbeEditCtrl.farmProbe.type !== 'http' && match === 'status'">
```

Comment:
Can the `oui-radio-toggle-group` be empty ? Wouldn't it be an issue ? Also, can you use `ngIf` instead? Hiding controls to the user but still having them is worrisome to me